### PR TITLE
844271: ASP-Need to add how to section to explain addNodes, updateNode, refreshNode, moveNodes and removeNodes method in TreeView documentation.

### PR DESCRIPTION
**Pull request by suba-1997**
### Bug description
ASP Need to add how to section to explain addNodes, updateNode, refreshNode, moveNodes and removeNodes method in TreeView documentation.

### Root cause
The mentioned methods are not documented earlier in ASP documentation.

### Solution description
We have included the separate page as User interactions in How To section.

### Reason for not identifying earlier
 * [] Guidelines not followed. If yes, provide which guideline is not followed.

 * [ ] Guidelines not given. If yes, provide which/who need to address.
    Tag label `update-guideline-coreteam` or `update-guideline-productteam`. 

 * [X] If any other reason, provide the details here. 
 Based on the customer raised queries, we have decided to document these methods in ASP.

#### Before:
image

#### After:
image

### Reviewer Checklist
* []  All provided information are reviewed and ensured.

### CODE REVIEW

File: ej2-asp-core-mvc/code-snippet/treeview/how-to/user-interactions-cs2/razor [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+                max-width: 500px;
+                margin: auto;
+                padding: 10px;
+            }
+</style>
+
+<script>
+    var treeObj = document.getElementById('treedata').ej2_instances[0];
+    document.getElementById("button1").addEventListener('click', function () {
+        // The node with id 21 is removed from the TreeView component.
```
Comment (kmkrish001) [inline ej2-asp-core-mvc/code-snippet/treeview/how-to/user-interactions-cs2/razor:32]: in comment mentioned as id 21 but in code it is 14

File: ej2-asp-core-mvc/code-snippet/treeview/how-to/user-interactions-cs2/tagHelper [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+</ejs-treeview>
+<ejs-button id="button1" cssClass="e-primary inline-element right" content="Remove parent"></ejs-button>
+<ejs-button id="button2" cssClass="e-primary inline-element right" content="Remove child"></ejs-button>
+
+<div id="display"></div>
+</div>
+<script>
+    var treeObj = document.getElementById('treedata').ej2_instances[0];
+    document.getElementById("button1").addEventListener('click', function () {
+        // The node with id 21 is removed from the TreeView component.
```
Comment (kmkrish001) [inline ej2-asp-core-mvc/code-snippet/treeview/how-to/user-interactions-cs2/tagHelper:13]: change comment

**Review by kmkrish001 — CHANGES_REQUESTED** [review]

File: ej2-asp-core-mvc/code-snippet/treeview/how-to/user-interactions-cs2/razor (hunk 1 above)
Comment (suba-1997) [reply ej2-asp-core-mvc/code-snippet/treeview/how-to/user-interactions-cs2/razor]: comment changed.

File: ej2-asp-core-mvc/code-snippet/treeview/how-to/user-interactions-cs2/tagHelper (hunk 2 above)
Comment (suba-1997) [reply ej2-asp-core-mvc/code-snippet/treeview/how-to/user-interactions-cs2/tagHelper]: comment changed.

**Review by kmkrish001 — APPROVED** [review]

**Pull request closed without merging**